packages/strategy-tests/src/lib.rs

```diff
@@ -1120,17 +1118,111 @@ impl Strategy {
+                                owner_id,
+                                *identity_nonce,
+                            );
+
+                            // Create `doc_type_count` doc types
+                            let mut doc_types = Value::Map(Vec::new());
+                            for _ in doc_type_count.clone() {
+                                match DocumentTypeV0::random_document_type(
+                                    params.clone(),
+                                    contract_id,
+                                    rng,
+                                    platform_version,
+                                ) {
+                                    Ok(new_document_type) => {
+                                        let mut doc_type_clone = new_document_type.schema().clone();
+                                        let name = doc_type_clone.remove("title").expect(
+                                            "Expected to get a doc type title in ContractCreate",
+                                        );
+                                        if let Value::Map(ref mut map) = doc_types {
+                                            map.push((
+                                                Value::Text(name.to_string()),
+                                                doc_type_clone,
+                                            ));
+                                        } else {
+                                            error!("doc_types is not a Value::Map as expected");
+                                        }
+                                    }
+                                    Err(e) => {
+                                        error!("Error generating random document type: {:?}", e)
+                                    }
+                                }
+                            }
+
+                            let contract_factory = match DataContractFactory::new(
+                                platform_version.protocol_version,
+                            ) {
+                                Ok(contract_factory) => contract_factory,
+                                Err(e) => {
+                                    error!("Failed to get DataContractFactory while creating random contract: {e}");
+                                    continue;
+                                }
+                            };
+
+                            let created_data_contract = match contract_factory.create(
+                                owner_id,
+                                *identity_nonce,
+                                doc_types,
+                                None,
+                                None,
+                            ) {
+                                Ok(contract) => contract,
+                                Err(e) => {
+                                    error!("Failed to create random data contract: {e}");
```

Comment:
I didn't know we had error! , not sure what it does.

Reply:
It's a tracing log, same as `info!`, so in the TUI it logs to `output.log`